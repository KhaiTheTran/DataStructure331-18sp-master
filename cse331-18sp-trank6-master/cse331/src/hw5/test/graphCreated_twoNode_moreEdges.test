# Test graphCreated_twoNode_moreEdges

# Create a graph
CreateGraph graphCreated_twoNode_moreEdges

# Add two nodes
AddNode graphCreated_twoNode_moreEdges xx
AddNode graphCreated_twoNode_moreEdges yy

# Add five edges
AddEdge graphCreated_twoNode_moreEdges xx xx xxxx
AddEdge graphCreated_twoNode_moreEdges xx yy xxyy9
AddEdge graphCreated_twoNode_moreEdges xx yy xxyy99
AddEdge graphCreated_twoNode_moreEdges yy xx yyxx
AddEdge graphCreated_twoNode_moreEdges yy yy yyyy

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
ListNodes graphCreated_twoNode_moreEdges
ListChildren graphCreated_twoNode_moreEdges xx
ListChildren graphCreated_twoNode_moreEdges yy