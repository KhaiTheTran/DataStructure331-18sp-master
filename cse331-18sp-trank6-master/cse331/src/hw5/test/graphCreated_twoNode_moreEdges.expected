# Test graphCreated_twoNode_moreEdges

# Create a graph
created graph graphCreated_twoNode_moreEdges

# Add two nodes
added node xx to graphCreated_twoNode_moreEdges
added node yy to graphCreated_twoNode_moreEdges

# Add five edges
added edge xxxx from xx to xx in graphCreated_twoNode_moreEdges
added edge xxyy9 from xx to yy in graphCreated_twoNode_moreEdges
added edge xxyy99 from xx to yy in graphCreated_twoNode_moreEdges
added edge yyxx from yy to xx in graphCreated_twoNode_moreEdges
added edge yyyy from yy to yy in graphCreated_twoNode_moreEdges

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing from yy
graphCreated_twoNode_moreEdges contains: xx yy
the children of xx in graphCreated_twoNode_moreEdges are: xx(xxxx) yy(xxyy9) yy(xxyy99)
the children of yy in graphCreated_twoNode_moreEdges are: xx(yyxx) yy(yyyy)